diagnostic.md

```diff
@@ -6,26 +6,29 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+// Add a resource to the routes file and a show action for that resource
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+The routes file   
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+// modifies the schema of our DB so that we can add/retreive things from it.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+/
+rails g scaffold thing with create a a full set of boilerplate files- model, db migration, controller and views
+
+rails g controller things will create a thing controller boilerplate file.
 ```
 
 What is the job of the `serializer`?
```

Comment:
Th serializer allows you to filter certain data out of your response. The data stil exists, it's just not shown in the response to users.